syncer/syncer.go

```diff
@@ -1934,184 +1933,54 @@ func (s *Syncer) handleRowsEvent(ev *replication.RowsEvent, ec eventContext) err
 }
 
 func (s *Syncer) handleQueryEvent(ev *replication.QueryEvent, ec eventContext, originSQL string) error {
-	if originSQL == "BEGIN" {
```

Comment:
removing this line will cause performance regression, because BEGIN is a hot path